### PR TITLE
Gulp4

### PR DESCRIPTION
**Pull request by 0dp**
I am hereby committing a PR for gulp4 update.

I had really hoped for more feedback from the community, but that's life.

@holger1411 @Thomas-A-Reinert @axlright and the rest of the UnderStrap team please review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,32 +27,25 @@
   },
   "homepage": "https://understrap.com",
   "dependencies": {
-    "bootstrap": "4.1.0",
-    "browser-sync": "^2.23.6",
+    "autoprefixer": "^8.6.2",
+    "bootstrap": "4.0.0",
```
Comment (pattonwebz) [inline package.json]: Should this be 4.1.0 to match the current master? Or maybe 4.1.1 to match BS4 latest?

Comment (0dp) [reply package.json]: Yeah should be `"bootstrap": "4.1.0"`

**Comment by Thomas-A-Reinert** [comment]
As already said - I had to take a few days off. Everything seems to work as expected for me.
Would love to have anyone else review and test before I merge it.

**Comment by ZacharyElkins** [comment]
@Thomas-A-Reinert I can test it this Saturday, but that’s a few days away.

**Comment by 0dp** [comment]
@ZacharyElkins, please do. This PR have been a long time coming and I really don't want it to go stale.

**Comment by Thomas-A-Reinert** [comment]
Sorry, but this conflicts with the gulp-rev as expected :/
Could you try to fix it asap please? Don´t really know how to fix this on my own. Would love to close this act as soon as possible..

**Comment by 0dp** [comment]
@Thomas-A-Reinert fixed!

**Comment by Thomas-A-Reinert** [comment]
Damn, the latest commit #722 effed it up again.. This is becoming a nightmare..
Could you?..

**Comment by 0dp** [comment]
sorted @Thomas-A-Reinert

**Comment by Thomas-A-Reinert** [comment]
Just noticed that package.json still/again contains "bootstrap": "4.0.0",

**Pull request merged**